.travis.yml

```diff
@@ -36,8 +36,10 @@ jobs:
       php: 7.3
       env: COMPOSER_FLAGS=""
       script:
+        - rm -rf vendor/
```

Comment:
You add this to avoid the following message when building Dockerfile ?
```
  [RuntimeException]                                                           
  Source directory /phpinsights/vendor/nesbot/carbon has uncommitted changes.  
```


That should do the trick :+1:

Reply:
hello @Jibbarth , this solves travis crashes, as we can see from the log [here](https://travis-ci.org/nunomaduro/phpinsights/jobs/574398249#L751), every time a tag was published composer install was failing for some reason from the vendor folder. = )